src/components/Register/Register.tsx

```diff
@@ -0,0 +1,597 @@
+                    )}
+                  </FormControl>
+                )}
+              />
+            </div>
+
+            <div className="form-input__register">
+              <Typography
+                align="left"
+                gutterBottom
+                sx={{
+                  fontSize: '16px',
+                  lineHeight: '24px'
+                }}>
+                Quận/Huyện
+                <Typography
+                  sx={{
+                    marginLeft: '2px',
+                    display: 'inline-block',
+                    color: 'red'
+                  }}>
+                  (*)
+                </Typography>
+              </Typography>
+              <Controller
+                render={({ field, fieldState: { error } }) => (
+                  <FormControl fullWidth required sx={{ mb: 2 }}>
+                    <InputLabel id="input-label">District</InputLabel>
+                    <Select
+                      labelId="demo-simple-select-label"
+                      id="district-select"
+                      label="district"
+                      {...field}
+                      fullWidth
+                      onChange={(event) => {
+                        field.onChange(event.target.value);
+                      }}>
+                      {districts.map((district) => (
+                        <MenuItem key={district.id} value={district.id}>
+                          {district.name}
+                        </MenuItem>
+                      ))}
+                    </Select>
+                    {error && (
+                      <FormHelperText sx={{ color: 'red' }}>
+                        {error?.message}
+                      </FormHelperText>
+                    )}
+                  </FormControl>
+                )}
+                name="district_id"
+                control={control}
+              />
+            </div>
+
+            <div className="form-input__register">
+              <Typography
+                align="left"
+                gutterBottom
+                sx={{
+                  fontSize: '16px',
+                  lineHeight: '24px'
+                }}>
+                Xã/Phường
+                <Typography
+                  sx={{
+                    marginLeft: '2px',
+                    display: 'inline-block',
+                    color: 'red'
+                  }}>
+                  (*)
+                </Typography>
+              </Typography>
+              <Controller
+                name="ward_id"
+                control={control}
+                render={({ field, fieldState: { error } }) => (
+                  <FormControl fullWidth required sx={{ mb: 2 }}>
+                    <InputLabel id="input-label">Ward</InputLabel>
+                    <Select
+                      labelId="demo-simple-select-label"
+                      id="ward-select"
+                      label="ward"
+                      {...field}
```

Comment:
```suggestion
                    <InputLabel id="input-label">Ward</InputLabel>
                    <Select
                      defaultValue=''
                      labelId="demo-simple-select-label"
                      id="ward-select"
                      label="ward"
                      {...field}
```

Sửa lại labelId, thêm defaultValue='' để tránh lỗi out of range